components/ExpertiseSelector.js

```diff
@@ -160,112 +35,330 @@ export default function ExpertiseSelector({ invitation, venueId, apiVersion, sho
+                return (
+                  <div className="keyphrase-container">
+                    <div className="d-flex">
+                      <input
+                        type="checkbox"
+                        className="checkbox-inline"
+                        id={`keyphrase-${index}`}
+                        disabled
+                      />
+                      <span className="inactive-keyphrase">
+                        <s>{keyphrase}</s>
+                      </span>
+                    </div>
+                    <Icon
+                      extraClasses={IconStyles.clickable}
+                      name="plus"
+                      tooltip="Add Keyphrase"
+                      onClick={() => {
+                        setKeyphrases({
+                          active: [...keyphrases['active'], keyphrase],
+                          inactive: keyphrases['inactive'].filter((k) => k !== keyphrase),
+                        })
+                        setIncludedKeyphrases([...includedKeyphrases, keyphrase])
+                      }}
+                    />
+                  </div>
+                )
+              })}
+
+              <div className="row keyphrase-container">
+                <input
+                  id="addkeyphrase"
+                  type="text"
+                  className="col-sm-7 form-control"
+                  placeholder="Add Keyphrase"
+                  onKeyPress={(e) => {
+                    if (e.key === 'Enter') {
+                      setKeyphrases({
+                        active: [...keyphrases['active'], e.target.value],
+                        inactive: keyphrases['inactive'],
+                      })
+                      setIncludedKeyphrases([...includedKeyphrases, e.target.value])
+                      e.target.value = ''
+                    }
+                  }}
+                />
+                <IconButton
+                  text="Add"
+                  name="plus"
+                  tooltip="Add Keyphrase"
+                  onClick={() => {
+                    var keyphrase = document.getElementById('addkeyphrase').value
+                    setKeyphrases({
+                      active: [...keyphrases['active'], keyphrase],
+                      inactive: keyphrases['inactive'],
+                    })
+                    setIncludedKeyphrases([...includedKeyphrases, keyphrase])
+                    document.getElementById('addkeyphrase').value = ''
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+
+          <div className="col-md-9 paper-container">
+            <div className="row column-header">
+              <h4 className="col-sm-10">Potential Paper Assignments</h4>
+            </div>
+            <div className="mb-3 ml-3">
+              Potential assignments from papers accepted in machine learning conferences from
+              2018 to 2023
```

Comment:
This text and these dates shouldn't be hardcoded, it could come from config vars or another source

Reply:
Thanks! This `div` might get removed in a future iteration. If not where do you suggest I get this? I couldn't find a specific `config` var in the `openreview-web` codebase, however I'm aware of `config` within the v1/v2 APIs being present.